src/app/api/services/dot-license/dot-license.service.spec.ts

```diff
@@ -1,80 +1,85 @@
-import { TestBed } from '@angular/core/testing';
+import { DOTTestBed } from '../../../test/dot-test-bed';
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs/Observable';
 import { LoginService } from 'dotcms-js/dotcms-js';
 import { DotRouterService } from '../dot-router/dot-router.service';
 import { ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
-import { ConfigParams, DotcmsConfig } from 'dotcms-js/core/dotcms-config.service';
+import { ConfigParams } from 'dotcms-js/core/dotcms-config.service';
 import { DotLicenseService } from './dot-license.service';
-
-@Injectable()
-class MockDotcmsConfigService {
-    getConfig(): Observable<ConfigParams> {
-        return Observable.of({
-            license: {
-                level: 100
-            },
-            disabledWebsockets: '',
-            websocketReconnectTime: 0,
-            websocketEndpoints: '',
-            websocketsSystemEventsEndpoint: '',
-            websocketBaseURL: '',
-            websocketProtocol: '',
-            menu: [],
-            paginatorRows: 0,
-            paginatorLinks: 0,
-            emailRegex: ''
-        });
-    }
-}
+import { ConnectionBackend, ResponseOptions, Response } from '@angular/http';
+import { MockBackend } from '@angular/http/testing';
 
 describe('DotLicenseService', () => {
-    let dotCmsConfig: DotcmsConfig;
-    let dotLicenseService: DotLicenseService;
-
     beforeEach(() => {
-        TestBed.configureTestingModule({
-            providers: [DotLicenseService, { provide: DotcmsConfig, useClass: MockDotcmsConfigService }]
-        });
-
-        dotCmsConfig = TestBed.get(DotcmsConfig);
-        dotLicenseService = TestBed.get(DotLicenseService);
+        this.injector = DOTTestBed.resolveAndCreate([DotLicenseService]);
+        this.dotLicenseService = this.injector.get(DotLicenseService);
+        this.backend = this.injector.get(ConnectionBackend) as MockBackend;
+        this.backend.connections.subscribe((connection: any) => (this.lastConnection = connection));
     });
 
     it('should return a false response because license is 100 = Community', () => {
-        let result: boolean;
-        dotLicenseService.isEnterpriseLicense().subscribe((res) => (result = res));
+        let result;
+        this.dotLicenseService.isEnterpriseLicense().subscribe((res) => (result = res));
+        this.lastConnection.mockRespond(
+            new Response(
+                new ResponseOptions({
+                    body: {
```

Comment:
You might want to create a util that you send the level and return you the object, to many duplicated code to mock this.